packages/entity-browser/src/modules/detailView/actions.js

```diff
@@ -44,3 +48,33 @@ export const setEntity = entity => ({
 export const submitForm = () => ({
   type: SUBMIT_FORM
 })
+
+export const loadRelationEntities = entityName => ({
```

Comment:
Nope this function loads all possible Entities for a specific relation. For ex.: for `relGender` it loads Gender(male) and Gender(female). This action gets fired if a select box gets focused.